src/spec.md

```diff
@@ -241,6 +255,9 @@ Webxdc apps run in a restricted environment, but the following practices are per
   e.g. with <https://babeljs.io>
 - `<a href="https://example.org/foo">` and other external links are blocked by definition;
   instead, embed content or use `mailto:` link to offer a way for contact
+- APIs that require user permission may not work, Geolocation, Camera, Microphone etc.
+- `<frame>` or `<iframe>` may not work as expected as used by some implementations for some features
+- `window.open()`, `alert()`, `prompt()`, `confirm()`, is known to not work on some implementations
```

Comment:
fun fact: in recent versions of dc desktop and electron `alert` and `confirm` work (they were not implemented before), but prompt does not and there is a message in the console that states that electron will never implement it.

Reply:
nice - so we can double check all systems at some point and strike alert() and confirm() from the list

that would be a great improvement, it is nice to have these tools at hand